client/src/renderer/components/Editor/index.jsx

```diff
@@ -38,15 +41,35 @@ export default class Editor extends Component {
     this.contentEditableDiv.current.innerHTML = "";
   }
 
+  //returns absolute values of caret's start/end positions
+  getCaretPos() {
+    const editor = document.getElementById("editor");
+    var positions = { pos1: 0, pos2: 0 };
+
+    var range = document.getSelection().getRangeAt(0);
+    var preSelectionRange = range.cloneRange();
+    preSelectionRange.selectNodeContents(editor);
+    preSelectionRange.setEnd(range.startContainer, range.startOffset);
+    positions.pos1 = preSelectionRange.toString().length;
+    positions.pos2 = positions.pos1 + range.toString().length;
+    console.log("Editor: ", positions.pos1, positions.pos2);
+    return positions;
```

Comment:
Might be easier to return an object, than build an object during the method. i.e.

```suggestion
    return { pos1, pos2 };
```

Reply:
fixed